Frontend/src/components/admin/CoopReviewPageNewCoopItem.vue

```diff
@@ -100,23 +100,11 @@ export default {
         status: "COMPLETED"
```

Comment:
The changes in this file fix a refresh issue where the Coops for an Admin to review would refresh before the Coop's status was actually updated